astropy/time/core.py

```diff
@@ -828,7 +829,12 @@ def _set_scale(self, scale):
                     break
 
             conv_func = getattr(erfa, sys1 + sys2)
-            jd1, jd2 = conv_func(*args)
+
+            # ignore warnings that are raised if nan are in inputs
+            with warnings.catch_warnings():
+                warnings.filterwarnings("ignore", message="invalid value")
+                warnings.filterwarnings("ignore", category=erfa.ErfaWarning)
```

Comment:
But what about correct warnings? I think that was the reason for @taldcroft original way to use `jd2=nan` and then replace with `jd2=0` before computations (refilling after). (For both cases, I also worry a bit about getting worse performance for the "standard" case with no bad data.

Reply:
This was largely to see the actual failures in the tests instead of errors due to CI warnings, not intended to be the actual solution...